packages/core/src/components/axes/zoom-bar.ts

```diff
@@ -128,6 +114,41 @@ export class ZoomBar extends Component {
 			// add value 0 to the extended domain for zoom bar area graph
 			this.compensateDataForDefaultDomain(zoomBarData, defaultDomain);
 
+			const initialZoomDomainInModel = this.model.get(
```

Comment:
what about calling them new and old?

Reply:
Good point, updated in https://github.com/carbon-design-system/carbon-charts/pull/793/commits/9c1323a6a6807c1c6e3469569bed5069fcbefd6f.
Thanks.